From 103967a7e0ac4ed6afc39b1b299af93e9d5ab8ff Mon Sep 17 00:00:00 2001
From: Lars Mueller <cobinja@yahoo.de>
Date: Wed, 4 Jan 2017 13:23:27 +0100
Subject: [PATCH] Fix hotcorner hover delay timeout removal This fixes a
 "Invalid or null source id used when attempting to run
 Mainloop.source_remove()" when leaving a hotcorner. The timeout is already
 destroyed by returning false in the callback, but we still knew about the
 timeout id and tried to remove the non-existing timeout.

---
 js/ui/hotCorner.js | 1 +
 1 file changed, 1 insertion(+)

Index: cinnamon/js/ui/hotCorner.js
===================================================================
--- cinnamon.orig/js/ui/hotCorner.js	2017-05-19 16:12:52.317560458 +0200
+++ cinnamon/js/ui/hotCorner.js	2017-05-19 16:12:52.309560576 +0200
@@ -323,6 +323,7 @@
                 }
             }
         }));
+        this.hover_delay_id = 0;
         return false;
     },
 
